### ISSUE TITLE
export 'Portal' (imported as 'Portal') was not found in '@reactour/utils'

### ISSUE THREAD

**Issue by Joaroise** [elrumordelaluz/reactour#503]
I use reactour in a project, and all gone perfectly till yesterday. 
Yesterday I download my project from git and I can’t make it works.
The complete error message says:
“Attempted import error: 'Portal' is not exported from '@reactour/utils' (imported as 'Portal').
ERROR in ./node_modules/@reactour/tour/dist/tour.esm.js 845:36-42
export 'Portal' (imported as 'Portal') was not found in '@reactour/utils' (possible exports: Observables, bestPositionOf, getInViewThreshold, getPadding, getRect, getWindow, inView, isHoriz, isOutsideX, isOutsideY, safe, smoothScroll, useElemRect, useIntersectionObserver, useRect)”

Comparing with one other installation of the project where it works, I have been able to find that the diference is between the @reactour/utils versions. Where works is 0.4.0 and where not is 0.4.6.
After I read something similar at the answer of other of the isues here, I tried to solve it adding  to package.json the field:
"overrides": {
    "@reactour/utils": "0.4.0"
  }
But it also not works.

Can you give me a solution or tell me where I could be wrong?

Thanks.

P.D: I pretty newer on programation world. I apologize if anything I say is not properly explained. I still have to much to learn.

**Comment by elrumordelaluz** [comment]
Hi @Joaroise, thanks for open the _Issue_.

Can you please provide the `@reactour/tour` are you using? The portal exported from `/utils` was the source of some packages versions and build issues and it was removed, allowing the user to implement (in case of need) their own through [Wrapper](https://github.com/elrumordelaluz/reactour/tree/main/packages/tour#wrapper-componenttype) prop. [Here](https://github.com/elrumordelaluz/reactour/blob/main/apps/web/pages/index.tsx#L41) is the implementation, using a custom Portal component in the demo.

The issue should be solved if you are using the last version of `/tour`, please let me know in any case.

**Comment by Allyson-Santana** [comment]
I had this same problem! @elrumordelaluz's resolution worked for me. Thank you so much!

**Comment by elrumordelaluz** [comment]
thank you @Allyson-Santana to share your (re)solutio!

updating into `v3.1.4` you still need the `resolution` workaround?

**Comment by Allyson-Santana** [comment]
No, thank you very much

**Comment by elrumordelaluz** [comment]
Since seems to be solved in `v3.1.4`, I am closing this _issue_, but feel free to re-open or create a new one in any case.

**Comment by kev2480** [comment]
I solved this issue by adding to my package.json:

I noticed something similar when installing (using yarn) "@reactour/tour": "2.13.0"

```
  "resolutions": {
    "@reactour/utils": "0.4.0",
    "@reactour/mask": "0.5.1",
    "@reactour/popover": "0.4.0"
  }
```
